server/src/application/application.module.ts

```diff
@@ -4,10 +4,11 @@ import { Application } from './application.entity';
 import { ApplicationController } from './application.controller';
 import { ApplicationService } from './application.service';
 import { UserModule } from '../user/user.module';
+import { CommentModule } from '../comments/comment.module';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([Application]), UserModule],
-  exports: [],
+  imports: [TypeOrmModule.forFeature([Application]), UserModule, CommentModule],
+  exports: [UserModule, CommentModule],
```

Comment:
Not sure why we'll have to do this?

Reply:
Since I deleted those from the DatabaseModule, here's the error I get if I don't export them (because I am importing only the ApplicationModule now)
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/102998728/205960075-148bad1a-6d27-450b-a46c-ad4a8c27d84b.png">

Reply:
I think that's because ApplicationService is used as a provider in the database module. That shouldn't be the case.

application module should export ApplicationService and only ApplicationModule should be imported in database module. @leon-fw

Reply:
Ah, that did it, thank you!